### PR TITLE
deps: chrome-launcher@0.10.7

### PR DESCRIPTION
**Pull request by paulirish**
#8331

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -132,7 +132,7 @@
   },
   "dependencies": {
     "axe-core": "3.2.2",
-    "chrome-launcher": "^0.10.5",
+    "chrome-launcher": "^0.10.6",
```
Comment (brendankenny) [inline package.json]: will you also delete the `resolutions` entry down on L164-166? yay!

**Comment by brendankenny** [comment]
is it possible the built js files aren't in 0.10.6? Or something else went wrong? Both CIs are still saying `Cannot find module 'chrome-launcher'`

**Comment by patrickhulce** [comment]
hm yeah something looks wrong there, there's no `dist/` folder

image

**Comment by patrickhulce** [comment]
heh @paulirish we were implicitly re-using the rimraf types from chrome-launcher in lighthouse-viewer :D

want to manually add `@types/rimraf`?

who knew a chrome-launcher bump would be so pesky

**Comment by brendankenny** [comment]
:arrows_counterclockwise::arrows_counterclockwise::arrows_counterclockwise:

**Pull request merged**